Automattic/wp-calypso
# Start enforcing & auto-fixing ESLint rule react/jsx-curly-brace-presence

#### Proposed Changes

* Sometimes this is flagged in code reviews and sometimes it isn't. For consistency, this PR proposes to enforce this rule and also benefit from the auto-fix feature of `eslint`. 

I opted for this rule config based on previous code review feedback:

![image](https://user-images.githubusercontent.com/6851384/195240504-bca2d897-2d79-4b1f-aeb2-4a06faeb93a1.png)

The rule homepage and full config options are [here](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-curly-brace-presence.md) 

#### Testing Instructions



* Checkout this branch and try to modify some JSX props to be like the **incorrect** format above. Verify they are caught by ESLint and can be auto-fixed. 

#### Pre-merge Checklist






Related to #
